doc/man7/OSSL_PROVIDER-serialization.pod

```diff
@@ -0,0 +1,99 @@
+=pod
+
+=head1 NAME
+
+OSSL_PROVIDER-serialization - OpenSSL serialization provider
+
+=head1 DESCRIPTION
+
+The OpenSSL serialization provider supplies the serialization for OpenSSL's
+asymmetric cryptography.
+
+=head2 Properties
+
+The implementations in this provider specifically have this property
+defined:
+
+=over 4
+
+=item "provider=serialization"
+
+=back
+
+It may be used in a property query string with fetching functions.
+
+It isn't mandatory to query for this property, except to make sure to get
+implementations of this provider and none other.
+
+The serialization algorithms in the default provider also have
+provider=serialization set.
+
+=over 4
+
+=item "type=parameters"
+
+=item "type=private"
+
+=item "type=public"
+
+=back
+
+These may be used in a property query string with fetching functions to select
+which data are to be serialized.  Either the private key material, the public
+key material or the domain parameters can be selected.
+
+=over 4
+
+=item "format=der"
+
+=item "format=pem"
+
+=item "format=text"
+
+=back
+
+These may be used in a property query string with fetching functions to select
+the serialization output format.  Either the DER, PEM and plaintext are
+currently permitted.
+
+=head1 OPERATIONS AND ALGORITHMS
+
+The OpenSSL default provider supports these operations and algorithms:
+
+=head2 Asymmetric Key Serializer
+
+In addition to "provider=serialization", some of these serializers define the
+property "fips=yes", to allow them to be used together with the FIPS
```

Comment:
Some? Which ones?

Reply:
Is a list required?  The default provider says the same thing without giving a list.

Reply:
Ok